website/content/api-docs/secret/kv/kv-v1.mdx

```diff
@@ -1,14 +1,16 @@
 ---
 layout: api
 page_title: KV - Secrets Engines - HTTP API
-description: This is the API documentation for the Vault KV secrets engine.
+description: This is the API documentation for the Vault KV secrets engine, version 1.
```

Comment:
Note that before this change, both KVV1 and KVV2 had the same descriptions and titles. I thought it best to update them to be different.